### ISSUE TITLE
Update query overrides catalog file ?

### ISSUE THREAD

**Issue by mark-jensen** [ontodev/robot#468]
I have an update query that works under normal conditions. However, I ran into trouble running it when not connected to the internet. A error is thrown stating that it can't load import files. I also noticed that running a simple select query works without having internet access. Commenting out the import declarations allows the update to complete normally. The update query I am using is a simple insert delete that changes version IRI.

The first few lines of the trace where the error arises: 

```
org.obolibrary.robot.ExceptionHelper - Missing exception ID: Could not load imported ontology:
<http://purl.obolibrary.org/obo/bfo.owl> Cause: purl.obolibrary.org 
org.semanticweb.owlapi.model.UnloadableImportException: Could not load imported ontology:
<http://purl.obolibrary.org/obo/bfo.owl> Cause: purl.obolibrary.org 
at uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.makeLoadImportRequest(OWLOntologyManagerImpl.java:1870)
```

My assumption is that something in the default behavior of update tries to resolve import IRIs, overriding the catalog file. To test: while connected to the internet, I changed the IRI of BFO in my import declaration, changed the IRI of a local copy of BFO, and fixed my catalog file to point to my local copy. The update query similarly failed with a slightly different error:

`Could not load imported ontology: <http://www.dummyurl.com/bfo.owl> Cause: Server returned HTTP response code: 400 for URL: http://www.dummyurl.com`

using version 1.4

**Comment by rctauber** [comment]
Hi @mark-jensen - thanks for bringing this up. Could you point me to the ontology that you are using and the UPDATE? I'd like to recreate this to try and figure out what's going on.

**Comment by jamesaoverton** [comment]
Hi @mark-jensen: I think that we found the problem. Thanks for the detailed report that helped us track it down. We'll try to fix it and get back to you.

**Comment by beckyjackson** [comment]
This was resolved by #471 and the fix is active in 1.4.1. 
Please reopen if you have any other issues with this.
